api/package.json

```diff
@@ -13,7 +13,8 @@
     "test":
       "NODE_ENV=test tap -J --cov --coverage-report=lcov --no-browser --reporter=spec --test-arg=--silent '{,!(node_modules)/**/}*.test.js'",
     "test-endpoints": "tap --reporter=spec 'endpoint-tests/*/**/*.js'",
-    "test-specific": "NODE_ENV=test tap --reporter=spec",
+    "test-specific":
+      "NODE_ENV=test tap --cov --coverage-report=lcov --no-browser --reporter=spec",
```

Comment:
Collect coverage even on specific tests.  This is useful (to me, anyway) when I'm writing tests, so I can see which lines I've covered and make sure that there are good assertions covering what the code is actually doing.